grpclog/internal/loggerv2.go

```diff
@@ -190,15 +211,36 @@ type LoggerV2Config struct {
 // The infoW, warningW, and errorW writers are used to write log messages of
 // different severity levels.
 func NewLoggerV2(infoW, warningW, errorW io.Writer, c LoggerV2Config) LoggerV2 {
-	var m []*log.Logger
 	flag := log.LstdFlags
 	if c.FormatJSON {
 		flag = 0
 	}
-	m = append(m, log.New(infoW, "", flag))
-	m = append(m, log.New(io.MultiWriter(infoW, warningW), "", flag))
-	ew := io.MultiWriter(infoW, warningW, errorW) // ew will be used for error and fatal.
-	m = append(m, log.New(ew, "", flag))
-	m = append(m, log.New(ew, "", flag))
+
+	infoLogW := infoW
+
+	// Use io.Discard instead of io.MultiWriter if all loggers at a given level
```

Comment:
no need of new lines in between sentences. Suggestion;
```
// Use io.Discard instead of io.MultiWriter when all loggers at a given level
// are set to io.Discard. Both this package and the standard log package have
// significant optimizations for io.Discard, which io.MultiWriter lacks (as of this writing).
```

Reply:
Changed